tetrateci/create_istio_release.sh

```diff
@@ -5,25 +5,48 @@ set -o pipefail
 
 # HACK : the github runner runs out of space sometimes so removing the 21 GB dotnet folder
 # Temporary thing, we should be moving to a custom runner instead.
+echo "Deleting /usr/share/dotnet to reclaim space"
 [ -d "/usr/share/dotnet" ] && sudo rm -rf /usr/share/dotnet
+echo "Deletetion complete"
+
+if [[ ${BUILD} == "fips" ]]; then
+    export GOLANG_VERSION=1.15.7b5
+    echo "Fetching FIPS compliant Go"
+    url="https://go-boringcrypto.storage.googleapis.com/go1.15.7b5.linux-amd64.tar.gz"
+    wget -O go.tgz "$url"
+    echo "cb08962897e3802cda96f4ee915ed20fbde7d5d85e688759ef523d2e6ae44851 go.tgz" | sha256sum -c -
+    sudo tar -C /usr/local -xzf go.tgz
+    rm go.tgz
+    export GOROOT=/usr/local/go
+    export PATH="$GOROOT/bin:$PATH"
+    echo "FIPS compliant Go installed"
+    go version
+else
```

Comment:
We need to build docker even for fips. Difference between fips and non fips is that, while creating package (istioctl), for non fips, we create it for different OSs and Arch. Whereas for fips, it would only be for Linux/AMD64.

Reply:
We can't specify in the manifest the architectures/os to be build for, it is harcoded [here](https://github.com/istio/release-builder/blob/654d06e868385a14c54d41d218448e304417f032/pkg/build/archive.go#L35) the only thing we can do is build but not push

Reply:
That's fine. Lets do that.